blazor/gantt-chart/filtering.md

```diff
@@ -263,9 +263,9 @@ You can filter the columns dynamically by using the [FilterByColumnAsync](https:
 
 @code{
     public SfGantt<TaskData> Gantt;
-    public void Filter()
+    public async void Filter()
     {
-        this.Gantt.FilterByColumnAsync("TaskName","startswith","Iden");
```

Comment:
We can add separate topic https://ej2.syncfusion.com/angular/documentation/grid/filtering/filtering#filter-grid-programmatically-with-single-and-multiple-values-using-method

Reply:
Currently Gantt chart doesn't have support for multiple value filter.